subprojects/packagefiles/libcurl/add_websockets.patch

```diff
@@ -0,0 +1,57 @@
+Only in curl-8.5.0-ws/: builddir
+Common subdirectories: curl-8.5.0/CMake and curl-8.5.0-ws/CMake
+Common subdirectories: curl-8.5.0/docs and curl-8.5.0-ws/docs
+Common subdirectories: curl-8.5.0/include and curl-8.5.0-ws/include
+Common subdirectories: curl-8.5.0/lib and curl-8.5.0-ws/lib
+Common subdirectories: curl-8.5.0/m4 and curl-8.5.0-ws/m4
+Common subdirectories: curl-8.5.0/meson and curl-8.5.0-ws/meson
+diff -p1 curl-8.5.0/meson.build curl-8.5.0-ws/meson.build
+*** curl-8.5.0/meson.build	2023-12-19 18:42:14.000000000 +0000
+--- curl-8.5.0-ws/meson.build	2024-03-09 06:55:35.835686735 +0000
+*************** foreach _opt : [
```

Comment:
what diff format is this? it does not look like default `diff` or `diff -u`

Reply:
I know very little about diff, but over [here](https://mesonbuild.com/Wrap-dependency-system-manual.html#diff-files) it states that it needs to be a `diff -p1`, so that's what i did :)

Reply:
Hang on, i see it doesn't need to be created with `diff -p1`.

Reply:
it's the `*****` that made me wonder, those are unrelated to `-p` level

Reply:
I was wrong, it does look  like `diff`, I've just grown very used to `diff -u`, and I think most people are these days. This is fine :)